snapcraft/commands/rbd-mirror.start

```diff
@@ -1,2 +1,7 @@
-#!/bin/sh
+#!/bin/bash
+
+. "${SNAP}/commands/common"
```

Comment:
any particular reason this was added?

Reply:
commands/common defines the `limits` shell func used below